x-pack/legacy/plugins/alerting/server/alerts_client.test.ts

```diff
@@ -119,7 +120,7 @@ describe('create()', () => {
       taskType: 'alerting:123',
       scheduledAt: new Date(),
       attempts: 1,
-      status: 'idle',
+      status: 'idle' as TaskStatus,
```

Comment:
shouldn't this be `status: TaskStatus.idle`?  I count 5 hits on `as TaskStatus` in the PR ...

Reply:
Argh, yeah, this got changed at a later point into an enum, and Typescript is WAY too forgiving of this stuff :/

Good catch, thanks